divvy/cloudfilebrowser.scss

```diff
@@ -527,36 +525,37 @@ the License.
         content: '\f15c';
         display: inline;
         font-family: 'FontAwesome';
-        font-size: 13px;
+        font-size: 1.5rem;
         margin-right: 8px;
     }
 
-    #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul li.filename:before { color: white }
-
     #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul li.foldername:before { content: '\f07b'; color: burlywood }
     #services-containers > div.provisioned > div.listTable > ul:first-child {
-        /*position: fixed;*/
-        background: linear-gradient(#555, #333);
-        border-left: 1px solid #777;
-        border-right: 1px solid #333;
-        box-shadow: inset 0 16px 2px rgba(255,255,255,.1), 0 -1px 0 rgba(255,255,255, .3), 0 -2px 0 rgba(0,0,0,.4), 0 1px 1px rgba(0,0,0,0.5);
-        color: #fff;
+        background: rgba(255,255,255,.1);
```

Comment:
Two things: use background-color, but also, why are we using an opacity here?

Reply:
All of the rgba values have an opacity for some reason. I kept it that way for consistency.

Reply:
Right. I should have asked if this needed to have transparency.